### ISSUE TITLE
bug(dev): build system does not support node > 16

### ISSUE THREAD

**Issue by rogueturnip** [codex-team/editor.js#2310]
Using the "next" branch I've been trying to build locally and I'm getting the following error

```
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
```
I've followed the example_dev directions with
yarn
yarn pull_tools
yarn _tools:make

on the last command I get the OSSL error.  I also get this with yarn run build:webpack:dev or yarn run devserver:start

Have I done this right to get a local setup going?

**Comment by neSpecc** [comment]
Node version?

**Comment by rogueturnip** [comment]
Running 18.4.0

**Comment by neSpecc** [comment]
Try in node 16

**Comment by rogueturnip** [comment]
` NODE_OPTIONS=--openssl-legacy-provider yarn _tools:make`
will allow it to build.  I'm chasing another bug on build right now

What is ERR_OSSL_EVP_UNSUPPORTED?
As mentioned in the [Node.js 17 release notes](https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V17.md#17.0.0), ERR_OSSL_EVP_UNSUPPORTED generally occurs when your application or one of its modules attempts to use an algorithm or key size which is no longer allowed by default with [OpenSSL 3.0](https://www.openssl.org/blog/blog/2021/09/07/OpenSSL3.Final/).

```
ERROR in ./src/styles/index.css (./node_modules/css-loader/dist/cjs.js!./src/styles/index.css)
Module build failed (from ./node_modules/css-loader/dist/cjs.js):
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './package.json' is not defined by "exports" in /Users/---/Code/editor.js/example/tools/text-variant-tune/node_modules/postcss/package.json
```

**Comment by neSpecc** [comment]
Our build system doesn't support Node 17 and 18 right now. We're working on the update.

**Comment by rogueturnip** [comment]
ok.  I'm just tracking the bugs seeing if I can figure out fixes..  The OSSL error is in the checklist module.

The next error is in the text-variant-tune module.

**Comment by neSpecc** [comment]
Resolved in #2300 by @talyguryn
